DotNet Examples/TestMonitor/DeleteResults/TestDataManagerDeleteExtensions.cs

```diff
@@ -0,0 +1,134 @@
+    {
+        private const string DeleteResultsPath = "nitestmonitor/v2/delete-results";
+        private const string DeleteResultPath = "nitestmonitor/v2/results";
+
+        /// <summary>
+        /// Deletes the test result
+        /// </summary>
+        /// <param name="testDataManager"></param>
+        /// <param name="configuration">HttpConfiguration containg the url and API key</param>
+        /// <param name="resultId">test result ID which needs to be deleted</param>
+        /// <param name="deleteSteps">A boolen representing whether to delete the associated steps or not</param>
+        /// <returns>Http reponse message after calling delete result API</returns>
+        /// <exception cref="ApiException"></exception>
+        public static HttpResponseMessage DeleteResultAsync(this TestDataManager testDataManager, IHttpConfiguration configuration, string resultId, bool? deleteSteps = true)
+        {
+            if (resultId == null)
+            {
+                throw new ApiException("Missing required parameter 'resultId' when calling ResultsApi->DeleteResultV2");
+            }
+            string path = $"{DeleteResultPath}/{resultId}?deleteSteps={deleteSteps}";
+
+            return CallDeleteResultApi(configuration, path);
+        }
+
+        /// <summary>
+        /// Deletes multiple test results
+        /// </summary>
+        /// <param name="testDataManager"></param>
+        /// <param name="configuration">HttpConfiguration containg the url and API key</param>
+        /// <param name="resultIds">test result Ids which needs to be deleted</param>
+        /// <param name="deleteSteps">A boolean representing whether to delete steps associated with the steps</param>
+        /// <returns>Http response message after calling the delete results API</returns>
+        /// <exception cref="ApiException"></exception>
+        public static HttpResponseMessage DeleteResultsAsync(this TestDataManager testDataManager, IHttpConfiguration configuration, List<string> resultIds, bool? deleteSteps = true)
+        {
+            if(resultIds.Count == 0)
+            {
+                throw new ApiException("ids is a required property for DeleteResultsRequest and cannot be null");
+            }
+
+            var bodyContent = new
+            {
+                Ids = resultIds.ToArray(),
+                DeleteSteps = deleteSteps
+            };
+
+            var body = GeneratePostRequestBody(bodyContent);
+
+            return CallDeleteResultsApi(configuration, body);
+        }
+
+        /// <summary>
+        /// calls the delete result API
+        /// </summary>
+        /// <param name="configuration">A HttpConfiguration object containing base url and api_key</param>
+        /// <param name="requestUri">Uri route</param>
+        /// <returns>Http reponse message after calling the API</returns>
+        private static HttpResponseMessage CallDeleteResultApi(IHttpConfiguration configuration, string requestUri)
```

Comment:
Name these methods better, it is not necessary to include these prefixes and rather use just verbs to explain the action

Reply:
updated the method names as raise post request and raise delete request in the HTTP client file